### PR TITLE
Supra Docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the Supra docs to apply Moonbeam standards and showcases a complete example of interacting with Supra. 

The termynal snippet has text that flows outside of the window, so I created a companion PR for mkdocs that updates the styling so that it wraps to the next line: https://github.com/papermoonio/moonbeam-mkdocs/pull/172

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If this page requires a disclaimer, I have added one

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

**Review by albertov19 — CHANGES_REQUESTED** [review]
This one is challenging to go through, but either way, it looks OK from my perspective.

Left some feedback

File: builders/integrations/oracles/supra.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+The Pull Oracle contract on Moonbeam is located at the following address:
+
+=== "Moonbeam"
+
+    ```text
+    {{ networks.moonbeam.supra.pull_oracle }}
+    ```
+
+=== "Moonbase Alpha"
```
Comment (themacexpert) [inline builders/integrations/oracles/supra.md:36]: I noticed that the contracts are verified for Moonbeam but not for Moonbase Alpha. Is it possible for the Supra team to verify these for Moonbase Alpha as well?

File: builders/integrations/oracles/supra.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+## Introduction {: #introduction }
+
+[Supra](https://supraoracles.com){target=\_blank} is a novel, high-throughput oracle and intralayer: a vertically integrated toolkit of cross-chain solutions (data oracles, asset bridges, automation network, and more) that interlink all blockchains, public (L1s and L2s) or private (enterprises), including Moonbeam.
+
+Supra provides decentralized oracle price feeds that can be used for on-chain and off-chain use cases such as spot and perpetual DEXes, lending protocols, and payment protocols.
+
+In this guide, you'll learn how to use Supra Oracles' price feeds to fetch price data in smart contracts on Moonbeam.
+
+--8<-- 'text/_disclaimers/third-party-content-intro.md'
```
Comment (themacexpert) [inline builders/integrations/oracles/supra.md:16]: IMHO, Is there a way to make the disclaimer smaller, like the bottom of the page? 

It doesn't have to be tiny but maybe a few sizes smaller ? 

Screenshot 2024-02-22 at 4 28 40 PM

**Review by themacexpert — DISMISSED** [review]
👍

File: builders/integrations/oracles/supra.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+3. Once the data has been written on-chain, the most recently published price feed data will be available in Supra's Storage contract
+
+You can fetch price data from Supra for any [available data pairs](https://supra.com/docs/data-feeds/data-feeds-index/){target=\_blank}.
+
+The Pull Oracle contract on Moonbeam is located at the following address:
+
+=== "Moonbeam"
+
+    ```text
+    {{ networks.moonbeam.supra.pull_oracle }}
```
[the remark this answers is not in the document]
Comment (eshaben) [reply builders/integrations/oracles/supra.md:33]: i didn't think so, but i double-checked, and we only do that on the DIA page. I'll reformat this to be a table and add links

**Comment by eshaben** [comment]
Moved the step-by-step instructions to a tutorial, because it's quite long. So I opted to try and leave the Supra page in the Builders section as more of a quick reference, here's what you need to know, and then link to the tutorial for a deep dive.

**Review by albertov19 — APPROVED** [review]
LGTM! Excellent idea on splitting the things into separate pages!

I also love the new scoped code snippet thingy 🤓

**Pull request merged**